variables.tf

```diff
@@ -337,7 +338,7 @@ variable "sqladmin_disable_polling" {
 }
 
 variable "storage_disable_polling" {
-  description = "Whetservicemanagement_disable_pollingher to disable polling for Storage API"
+  description = "Whether to disable polling for Storage API"
   type        = bool
```

Comment:
Nit since this is part of the PR - should put the default default first, type second, as per the other formats.